client/components/grand-perspective/clipboard.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+/* globals document */
+
+const _ = require('lodash');
+
+/* Browsers place restrictions on how javascript can access the system clipboard,
+ * so some slight of hand is required to enable a copy function.
+ *
+ * We must create a <textarea> that is added to the document only momentarily, and
+ * from that select and copy the inner text.
+ *
+ * See http://stackoverflow.com/a/30810322 for context. */
+const copy = (text) => {
```

Comment:
Of dubious importance/interest to test. Failure cost is so low and the testing would have to be tightly coupled to implementation.